### DISCUSSION TITLE
FSP output

### DISCUSSION THREAD

**Discussion by nanguaxiaofendui** [idaholab/moose#23692] [Q&A General]
Hello all,
Why do I use different FSP preprocessing options (additive, multiplicative, schur), but using -ksp_view_pmat are jacobian matrix output?

**Comment by GiudGiud** [comment]
Hello

Field splits are not just a different preconditioning matrix, they are also a sequence of steps to solve the linear systems.
This isnt shown by view_pmat

Guillaume
